src/runtime-rs/crates/resource/src/cpu_mem/initial_size.rs

```diff
@@ -105,7 +105,7 @@ impl InitialSizeManager {
             hv.cpu_info.default_vcpus = self.resource.vcpu as i32
```

Comment:
Don't you need to also do the same here?

Reply:
We have discussed a lot for this point in the original PR and decided not to plus the default_vcpus for now. The reasons are:

- Except for those io operations, the kata-agent process does not consume many resources when the container is running.  But in some recent discussions, we have seen that we are trying to skip kata-agent when doing io forwarder to improve performance and stability.
- Doing so will bring us more opportunities to optimize the overall resource management of the host. Including performance, resource overselling, etc. 
- The default value is very hard to manage, not only for the users but also for runtime-rs.

So let's see what happens when we do this. Do you think it's ok?

Reply:
Yes, totally okay.  Thanks!